packages/lesswrong/components/review/PostInteractionStripe.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import classNames from 'classnames';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    position: "absolute",
+    left: 0,
+    top: 0,
+    height: "100%",
+    width: 6,
+    background: theme.palette.grey[405],
+  },
+  bigUpvote: {
+    background: theme.palette.primary.dark
+  },
+  smallUpvote: {
+    background: theme.palette.primary.light
+  },
+  bigDownvote: {
+    background: theme.palette.error.dark
+  },
+  smallDownvote: {
+    background: theme.palette.error.light
+  },
+  readPost: {
+    background: theme.palette.grey[405]
+  }
+});
+
+const votePrefix = `You previously gave this post `
+
+const interactionLabels: AnyBecauseTodo = {
+  'bigDownvote': `${votePrefix}a strong (karma) downvote`,
+  'smallDownvote': `${votePrefix}a (karma) downvote`,
+  'smallUpvote': `${votePrefix}a (karma) upvote`,
+  'bigUpvote': `${votePrefix}a strong (karma) upvote`,
+  'readPost': `You have read this post`
+}
+
+export const PostInteractionStripe = ({classes, post}: {
+  classes: ClassesType,
```

Comment:
should not have explicit : JssStyles return type, and classes prop in the component should be typed classes: ClassesType<typeof styles> to get type inference for the available class names